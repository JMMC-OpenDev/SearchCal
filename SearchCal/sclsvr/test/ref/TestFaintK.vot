version https://git-lfs.github.com/spec/v1
oid sha256:958ab6a40b9282a7e59262f4cfe09f6328a32324a3ebde24feb25d8fb98d7721
size 2079606
