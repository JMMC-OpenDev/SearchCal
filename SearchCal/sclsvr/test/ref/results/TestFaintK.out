version https://git-lfs.github.com/spec/v1
oid sha256:10065f121c1ff8799e39cabc0b4a6b33b99a2c4a6ca3fb0c432f1b703436bc70
size 1711116
